CMakeLists.txt

```diff
@@ -63,6 +63,12 @@ endif()
 
 if(UTF8PROC_ENABLE_TESTING)
   enable_testing()
+  file(MAKE_DIRECTORY data)
+  set(UNICODE_VERSION 13.0.0)
+  file(DOWNLOAD http://www.unicode.org/Public/${UNICODE_VERSION}/ucd/NormalizationTest.txt data/NormalizationTest.txt SHOW_PROGRESS)
+  file(DOWNLOAD http://www.unicode.org/Public/${UNICODE_VERSION}/ucd/auxiliary/GraphemeBreakTest.txt data/GraphemeBreakTestOrg.txt SHOW_PROGRESS)
+  file(REMOVE data/GraphemeBreakTestOrg.txt)
```

Comment:
Why not download as `data/GraphemeBreakTest.txt`?

Reply:
Is the same name as in the Makefile and data\Makefile

Reply:
Woodhead error, removed a line `    execute_process(COMMAND bash -c "cat data/GraphemeBreakTestOrg.txt | /usr/bin/perl -pe 's,÷,/,g;s,×,+,g'" OUTPUT_FILE data/GraphemeBreakTest.txt)
`